pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -389,6 +396,118 @@ spec:
           value: "{{ .Value }}"
 {{- end }}
 {{- end }}
+{{- if .externalEtcd }}
+---
+kind: EtcdadmCluster
+apiVersion: etcdcluster.cluster.x-k8s.io/v1beta1
+metadata:
+  name: "{{.clusterName}}-etcd"
+  namespace: "{{.eksaSystemNamespace}}"
+spec:
+  replicas: {{.externalEtcdReplicas}}
+  etcdadmConfigSpec:
+    etcdadmBuiltin: true
+    format: {{.format}}
+{{- if .etcdNtpServers }}
+    ntp:
+      enabled: true
+      servers: {{ range .etcdNtpServers }}
+      - {{ . }}
+      {{- end }}
+{{- end }}
+    cloudInitConfig:
+      version: {{.externalEtcdVersion}}
+      installDir: "/usr/bin"
```

Comment:
Can you add `etcdReleaseURL` here similar to this https://github.com/aws/eks-anywhere/blob/main/pkg/providers/vsphere/config/template-cp.yaml#L562-L564

Reply:
Fixed